src/pages/workshops/coreboot-july-2022.tsx

```diff
@@ -0,0 +1,47 @@
+import { FunctionComponent } from 'react'
+
+import { Layout, GatedResourceLayout } from '@components'
+
+export const Workshops: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'Coreboot + Sourcegraph Workshop',
+            description: '',
```

Comment:
If the description value is blank we can omit the whole line, it's not needed :)